drivers/bridge/bridge.go

```diff
@@ -674,6 +675,7 @@ func (d *driver) createNetwork(config *networkConfiguration) error {
 	bridgeAlreadyExists := bridgeIface.exists()
 	if !bridgeAlreadyExists {
 		bridgeSetup.queueStep(setupDevice)
+		config.ifaceManaged = true
```

Comment:
Wouldn't this code mark as `managed` also the libnetwork created `docker0` bridge ?